src/main/kotlin/view/ResultView.kt

```diff
@@ -56,17 +56,27 @@ object ResultView {
     }
 
     private fun showDealerGameResult(dealer: Participant, players: List<Participant>) {
-        var win = 0
-        var lose = 0
-        var draw = 0
+        val gameResult = GameResult()
         players.forEach { player ->
-            when (player.gameResultState) {
-                GameResultState.WIN -> lose++
-                GameResultState.LOSE -> win++
-                GameResultState.DRAW -> draw++
-            }
+            gameResult.update(player.gameResultState)
+        }
+        println("${dealer.name}: $gameResult")
+    }
+
+    fun showGameProfitResult(dealer: Dealer, players: List<Participant>) {
+        println("\n## 최종 수익")
+        showGameDealerProfitResult(dealer, players)
+        players.forEach { player ->
+            println("${player.name}: ${player.profit}")
+        }
+    }
+
+    private fun showGameDealerProfitResult(dealer: Dealer, players: List<Participant>) {
+        var dealerProfit = 0.0
+        players.forEach { player ->
+            dealerProfit += player.profit
         }
```

Comment:
```suggestion
        val dealerProfit = players.sumOf(Participant::profit)
```
sumOf와 메서드참조를 활용하면, 코드를 깔끔하게 작성할 수 있어요

Reply:
이렇게 수정했다가 Paricipant의 프로퍼티 수를 줄이면서 profit이 없어져서 기존처럼 forEach문으로 하게 되었습니다.